volumes/ecocounter/tables/counts.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE ecocounter.counts (
+    flow_id numeric REFERENCES ecocounter.locations (flow_id),
+    datetime_bin timestamp NOT NULL,
+    volume smallint,
+    UNIQUE(flow_id, datetime_bin)
+);
+
+CREATE INDEX ON ecocounter.counts (datetime_bin);
```

Comment:
probably an index on flow_id as well

Reply:
Good catch. I kinda assumed this was happening implicitly already like for a primary key, but I guess not:

> Since a DELETE of a row from the referenced table or an UPDATE of a referenced column will require a scan of the referencing table for rows matching the old value, it is often a good idea to index the referencing columns too. Because this is not always needed, and there are many choices available on how to index, declaration of a foreign key constraint does not automatically create an index on the referencing columns.

from [here](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-FK)